src/Microsoft.DotNet.Web.ProjectTemplates.1.x/content/StarterWeb-CSharp/Company.WebApplication1.csproj

```diff
@@ -50,17 +50,17 @@
     <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite.Design" Version="1.1.4" PrivateAssets="All" Condition=" '$(IndividualAuth)' == 'True' AND '$(UseLocalDB)' != 'True' " />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="1.1.4" Condition=" '$(IndividualAuth)' == 'True' AND '$(UseLocalDB)' == 'True' " />
     <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer.Design" Version="1.1.4" PrivateAssets="All" Condition=" '$(IndividualAuth)' == 'True' AND '$(UseLocalDB)' == 'True' " />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="1.1.2" PrivateAssets="All" Condition="'$(IndividualAuth)' == 'True'" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="1.1.4" PrivateAssets="All" Condition="'$(IndividualAuth)' == 'True'" />
```

Comment:
Why 4? Last release was 1.1.2

Reply:
For consistency with the EF runtime, which is going to be releasing a 1.1.4 patch. For some reason that no one could quite remember, we didn't publish a 1.1.3 version of EF tools even though there is a 1.1.3 EF runtime.

Reply:
EF Tools 1.x is in a different repo than EF 1.x. Their versions are out-of-sync because they were released out-of-sync.

@divega @ajcvickers Are you OK unifying EF and EF Tools in the next 1.0.x and 1.1.x patch?

Reply:
@bricelam versioning is a highly sensitive subject :smile: Why don't you tell us what you think the pro and cons are?

FWIW, from the perspective of a customer trying to reason about EF Core versions, I don't know that the repo where the 1.x tools leave being different matters too much.

Reply:
lol, I don't care either way. Just wanted to make sure nobody else did either.